internal/planner/planner.go

```diff
@@ -593,32 +539,26 @@ func (p *Planner) RunRequest(
 	return p.executeRequest(ctx, planNode)
 }
 
-// RunSubscriptionRequest plans a request specific to a subscription and returns the result.
-func (p *Planner) RunSubscriptionRequest(
-	ctx context.Context,
-	request *request.Select,
-) (result []map[string]any, err error) {
-	planNode, err := p.makePlan(request)
+// MakeSelectionPlan makes a plan for a single selection.
+//
+// Note: Caller is responsible to call the `Close()` method to free the allocated
+// resources of the returned plan.
+//
+// @TODO {defradb/issues/368}: Test this exported function.
```

Comment:
todo: This todo is incorrect - this is not a public function and does not require direct testing - if it is not used by us it can be deleted.

Reply:
Removed